src/backend/sev/snp/launch/linux.rs

```diff
@@ -100,58 +99,55 @@ impl<'a, T: Id> Command<'a, T> {
 /// Initialize the SEV-SNP platform in KVM.
 #[derive(Default)]
 #[repr(C, packed)]
-pub struct Init {
+pub struct Init2 {
+    /// initial value of features field in VMSA
+    vmsa_features: u64,
     /// Reserved space, must be always set to 0 when issuing the ioctl.
-    flags: u64,
+    flags: u32,
+    /// maximum guest GHCB version allowed
+    ghcb_version: u16,
+    pad1: u16,
+    pad2: [u32; 8],
 }
 
-impl Init {
+impl Init2 {
```

Comment:
Why the name change?

Reply:
They added a new ioctl, `KVM_SEV_INIT2`, for starting the launch measurement. Previously we just used `KVM_SEV_INIT`, but that's no longer recommended(/possible?) for SNP. The name change reflects the name of the ioctl.